src/main/java/Command.java

```diff
@@ -0,0 +1,15 @@
+import java.io.IOException;
+
+public abstract class Command {
+    private boolean exit = false;
```

Comment:
Perhaps consider using a better name that sounds like a boolean for this variable?